src/sentry/static/sentry/app/components/pagination.tsx

```diff
@@ -64,19 +75,17 @@ export default class Pagination extends React.Component {
         <div className="btn-group pull-right">
           <a
             onClick={() => {
-              onCursor(links.previous.cursor, path, query, -1);
+              callIfFunction(onCursor, links.previous.cursor, path, query, -1);
             }}
             className={previousPageClassName}
-            disabled={links.previous.results === false}
```

Comment:
Do we have any CSS that targets the `disabled` attribute?

Reply:
Not that I can see on issues or in settings.